.github/workflows/ubuntu-gcc.yml

```diff
@@ -0,0 +1,44 @@
+name: Ubuntu GCC
+
+# Controls when the action will run.
+on:
+    # Triggers the workflow on push or pull request events but only for the develop branch
+    push:
+        branches: [Preview4_0]
+    pull_request:
+        branches: [Preview4_0]
+
+    # Allows you to run this workflow manually from the Actions tab
+    workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+    # This workflow contains a single job called "build"
+    build:
+        strategy:
+            fail-fast: false
+            matrix:
+                os: [ ubuntu-18.04, ubuntu-20.04 ]
+
+        # The type of runner that the job will run on
+        runs-on: ${{ matrix.os }}
+
+        # Set job name
+        name: OS ${{ matrix.os }} GCC
+
+        # Steps represent a sequence of tasks that will be executed as part of the job
+        steps:
+            # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+            - uses: actions/checkout@v2
+
+            - uses: lukka/get-cmake@latest
+
+            - name: Install Ubuntu Dependencies
+              run: sudo apt-get install -y libgtk-3-dev libopenal-dev libxxf86vm-dev
```

Comment:
Is this enough? Seems like a way shorter list than the one in the Travis file?

Reply:
It seems to build for Ubuntu, but I can add further dependencies if they're needed: https://github.com/Ragora/Torque3D/actions

Reply:
It might be a difference in build vs run, I think I experienced that when running T3D in Docker